src/WorkItems/GitHubClientWrapper.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Octokit;
+using Octokit.Internal;
+
+namespace Microsoft.WorkItems
+{
+    /// <summary>
+    /// Represents a GitHub project in which work items can be filed.
+    /// </summary>
+    public class GitHubClientWrapper : FilingClient
```

Comment:
>GitHubClientWrapper [](start = 17, length = 19)

Yes, I see your problem, that Octokit already defines "GitHubClient" so you need another name. But how about `GitHubFilingClient` (a natural name for a class derived from `FilingClient`) and `AzureDevOpsFilingClient`?

As the name stands, I mentally stripped of the word "Wrapper" and tried to understand why you would need to "wrap" one of our own classes. It took me a few seconds to see that you were avoiding a conflict with someone else's class. Whereas if you'd named it `GitHubFilingClient` I wouldn't even have flinched.

Reply:
Yes.

---
In reply to: [381451103](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381451103) [](ancestors = 381451103)